code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.kt

```diff
@@ -146,7 +146,11 @@ class EdgeBridgeExtensionTests {
                 "__adobe" to mapOf(
                     "analytics" to mapOf(
                         "linkName" to "action name",
-                        "linkType" to "other"
+                        "linkType" to "other",
+                        "cp" to "foreground",
+                        "contextData" to mapOf(
+                            "a.AppID" to "null"
```

Comment:
Can you set this to an actual test value instead of the String "null" here. Something like "App 1.0 (2)"?